<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>523</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QRadioButton" name="Pulse">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>0</y>
      <width>20</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>72</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>541</width>
      <height>201</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <property name="iconSize">
     <size>
      <width>15</width>
      <height>15</height>
     </size>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Stream</string>
     </attribute>
     <widget class="QPushButton" name="Play">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>121</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>FreeSerif</family>
        <pointsize>36</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string> â–º</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Stop">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>121</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>FreeSerif</family>
        <pointsize>36</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>| |</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="path">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>80</y>
        <width>241</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="setaudiosource">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>110</y>
        <width>241</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Set audio source</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>20</y>
        <width>66</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Mean :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>50</y>
        <width>66</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Variance :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="setryhtmvariable">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>16</y>
        <width>91</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Set live BPM</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>-120</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="meanvariable">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>20</y>
        <width>60</width>
        <height>27</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>8</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="variancevariable">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>50</y>
        <width>60</width>
        <height>27</height>
       </rect>
      </property>
      <property name="minimum">
       <number>50</number>
      </property>
      <property name="maximum">
       <number>500</number>
      </property>
      <property name="value">
       <number>150</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>BPM</string>
     </attribute>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>121</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>Find precise BPM</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>50</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Precise BPM :</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>100</y>
        <width>321</width>
        <height>23</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="BPMlineedit">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>40</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Metro</string>
     </attribute>
     <widget class="QPushButton" name="Metrostate">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>121</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>Start 
Metronome</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="BPM">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>50</y>
        <width>71</width>
        <height>27</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>300</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="Measure">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>90</y>
        <width>71</width>
        <height>27</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>13</number>
      </property>
     </widget>
     <widget class="QPushButton" name="ModifyMetronome">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>121</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>Modify 
Metronome</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>50</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string> Wanted BPM :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>90</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Measure :</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="Phonon::VolumeSlider" name="volumeSlider">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>240</y>
      <width>20</width>
      <height>201</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="muteVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Phonon::SeekSlider" name="seekSlider">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>216</y>
      <width>561</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Tag">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>236</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="Bigtemp">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>0</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>85</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>85</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>159</red>
          <green>158</green>
          <blue>158</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>72</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>Bigtemp</zorder>
   <zorder>Pulse</zorder>
   <zorder>tabWidget</zorder>
   <zorder>volumeSlider</zorder>
   <zorder>seekSlider</zorder>
   <zorder>Tag</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>599</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Phonon::SeekSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/seekslider.h</header>
  </customwidget>
  <customwidget>
   <class>Phonon::VolumeSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/volumeslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
